test/mailers/user_mailer_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserMailerTest < ActionMailer::TestCase
+  # test "the truth" do
```

Comment:
This test doesn't seem to do anything.  Either add a test or delete the file if you can.